index.html

```diff
@@ -18,37 +19,8 @@
       <canvas id="model-canvas">Model canvas</canvas>
     </div>
     <br>
-  <!-- Incident &theta;: <input id="slider_incidentTheta" type="range" min="0" max="90" step="1" value="45"/> <output id="output_incidentTheta"></output>
-  <br>
-  Incident &phi;: <input id="slider_incidentPhi" type="range" min="-180" max="180" step="1" value="0"/> <output id="output_incidentPhi"></output>
-  <br>
-  Camera Rotation: <input id="slider_camRot" type="range" min="-180" max="180" step="1" value="0"/>  -->
 	</body>
 
-<!--
-   -<script type="vertex" id="phong.vert">
-   -    #version 300 es
-   -    layout (location=0) in vec4 model_position;
-   -    layout (location=1) in vec3 color;
-   -    layout (location=2) in vec4 model_normal;
-   -
-   -    uniform mat4 u_m;
-   -    uniform mat4 u_v;
-   -    uniform mat4 u_p;
-   -    
-   -    out vec3 vColor;
-   -    //out vec4 world_position;
-   -    out vec4 world_normal;
-   -
-   -    void main() {
-   -        vColor = color;
-   -        gl_Position = u_p * u_v * u_m * model_position;
-   -        //world_normal = u_m * model_normal;
-   -        world_normal = model_normal; //DEBUG ONLY.
-   -    }
-   -  </script>
-   -->
-
   <script type="vertex" id="lobe.vert">#version 300 es
```

Comment:
Can we get these shaders moved out of index.html? Maybe load them in using AJAX instead?